tests/integration/conftest.py

```diff
@@ -147,7 +147,7 @@ async def app_fixture(
     application = await model.deploy(
         charm, resources=resources, application_name=app_name, config=app_config, series="focal"
     )
-    await model.wait_for_idle()
+    await model.wait_for_idle(raise_on_error=False)
```

Comment:
Curios what the reason for this is please? If an error occurs can the tests keep going?

Reply:
I'll remove this later but as a note: it can happen that some charms reach an error state and recover soon after. This happens for the redis-k8s charm for example. So this could lead to us retrying CIs without needing it.

Reply:
Perhaps worth giving that feedback to the owner of the redis charm? It doesn't sound like the charm should enter error state

Reply:
I propose to handle that with https://warthogs.atlassian.net/browse/ISD-1001.